src/Servers/Kestrel/Core/src/Internal/Infrastructure/ITimeoutControl.cs

```diff
@@ -26,5 +26,6 @@ internal interface ITimeoutControl
         void StartTimingWrite();
         void StopTimingWrite();
         void BytesWrittenToBuffer(MinDataRate minRate, long count);
+        long GetWriteTimingTimeoutTimestamp();
```

Comment:
😵‍💫

Reply:
I know this is internal, but let's rename this to `GetWriteTimeoutTimestamp()`.